CLAUDE.md

```
@@ -0,0 +1,210 @@
+| Object Storage | MinIO SDK |
+| Email | Resend |
+| OAuth | Google Auth SDK |
+| Client UI | Avalonia 12, CommunityToolkit.Mvvm |
+| Client DI | Autofac 9.1 |
+| Testing | xUnit v3, FluentAssertions, Moq, coverlet |
+
+Central package management via `Directory.Packages.props` (`ManagePackageVersionsCentrally=true`).
+
+## Building and Running
+
+### Prerequisites
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)